pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -4557,6 +4677,7 @@ private void areaPropsContextMenuItem_Click(object sender, EventArgs e)
         {
             switch (ContextMenuGraphSummary.Type)
             {
+                case GraphTypeSummary.abundance:
```

Comment:
This seems a bit strange that the abundance graph should share a properties form with the comparison forms.